File: java_work/ex08/src/ex01/Ex01.java
package ex01;

public class Ex01 {

    public static void doA() {
        int i = 0;
        while (i < 10) {
            int j = 0;
            while (j <= i) {
                System.out.print("*");
                j++;
            }
            System.out.println();
            i++;
        }
    }
    public static void main(String[] args) {
        doA();
    }
}

File: js_work/240820/ex01.js
// function doA() {
//     alert('누름');
// }

var heading = document.querySelector("#heading");
// console.log(heading);
var btn = document.querySelector("#btn");

heading.onclick = function() {
    heading.style.color = "red";
    heading.innerHTML = "글자도 바꿈";
    heading.style.backgroundColor = "blue";
    heading.style.textAlign="center";
    heading.style.fontSize = "100px";
};

btn.onclick = function() {
    var name = prompt('당신의 이름은?', '이름');
    alert(name);
}